01_fib/baijie/fib_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := strconv.Quote("1\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main(), expected %v, actual %v", expected, actual)
```

Comment:
Don't capitalize error strings (from [https://github.com/anz-bank/go-course/wiki/Lab-1:-Collected-Code-Review-Comments](url), see also [https://github.com/golang/go/wiki/CodeReviewComments#error-strings](url)

Reply:
Done

Reply:
@alextmz Those links you provided are broken - you have the text and the link backwards. Also, those comments about error strings refer to strings in `error`s, not the output of `log.Printf`, `fmt.Printf`, etc. That is because errors are often wrapped and included within other output so capitals can end up in the middle of the line. For `t.Errorf` which is a final output, starting with a capital is just fine.